### PR TITLE
tess_updates

### PR DESCRIPTION
**Pull request by tesscoulthard**
main changes include:
- updating tenses to present tense where required
- removing additional capitalisation
- added the word "far" to the last line to match the metre of the second line (the one it rhymes with).

Only other notes are: 
On the line "Hector moves another paw." - I don't know if I would describe a dinosaurs foot as a paw, I would probably say foot or leg (but I didn't want to make the change because paw actually sounds pretty cute).

### CODE REVIEW

**Comment by dbertella** [comment]
@mslobster

File: src/content/index.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ path: /
 ---
 
 Hector is a little dinosaur  
-Who lives in the forest.  
+who lives in the forest.  
 He spends all his days eating flowers  
-And rest.
+and resting.
```
Comment (MsLobster) [inline src/content/index.md:11]: Oh makes sense! But I might think of something else to make it rhyme with forest

File: src/content/index.md [hunk 2]
```diff
@@ -108,58 +108,58 @@ But the snail is much faster.
...
 ![page 15](../images/page15.png)
 
 ---
 
-When a little mouse  
-Comes to offer him a nut.  
-“I’m small and fast, I can give you more.  
+Then a little mouse  
+comes to offer him a nut.  
+“I’m small and fast, and I can get you more nuts.
```
Comment (MsLobster) [inline src/content/index.md:129]: Ha! Is it indispensable to add 'nuts' after 'more'? I liked that it rhymed with 'protector'!

**Review by MsLobster — COMMENTED** [review]
Happy with most of your amends!
I might review a couple of them to see if I can think of something else to make it rhyme.
I also need to decide if the capitalisation will remain or not.

Thanks a lot Tess!

Aure

**Comment by dbertella** [comment]
I merged this one, we can add more improvements later. Thanks for the contribution @tesscoulthard!

**Comment by tesscoulthard** [comment]
YAY! Thanks guys - always happy to help :)

Am I allowed to share to italian version with a friend of mine. She is learning
Italian for her girlfriend and I think it would be a good learning tool?
xxx
ALL THE BEST,TESS

TESS COULTHARDACCOUNT MANAGER+44 (0)7983 612601
FOR URGENT REQUESTS, PLEASE CC OUR TECH SUPPORT INBOX:
PRODUCTIONSUPPORT@WEGOTPOP.COM

WE GOT POP LTD
UK HEADQUARTERS: THE TEA BUILDING, 56 SHOREDITCH HIGH STREET, LONDON, E1 6JJUS
HEADQUARTERS: 42 WEST ST., BROOKLYN 11222






On Sun, Mar 1, 2020 10:12 PM, Daniele Bertella notifications@github.com  wrote:
I merged this one, we can add more improvements later. Thanks for the
contribution @tesscoulthard!

**Comment by dbertella** [comment]
Of course you can! it's a public website now!

**Pull request merged**